### PR TITLE
FAB replacement for investigations and dashboards

### PR DESCRIPTION
**Pull request by abijahcalderon**
### Proposed changes
- conditionally replaced creation FAB for investigations 
- conditionally replaced create/import FAB for dashboards

### Checklist

- [X] I consider the submitted work as finished
- [X] I tested the code for its functionality
- [X] I wrote test cases for the relevant uses case (coverage and e2e)
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-front/tests_e2e/dashboard/createDashboard.spec.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ test('Create a new dashboard page', async ({ page }) => {
   const dashboardDetailsPage = new DashboardDetailsPage(page);
   const dashboardForm = new DashboardFormPage(page);
   await page.goto('/dashboard/workspaces/dashboards');
-  await dashboardPage.openButtonModal().hover();
+  // await dashboardPage.openButtonModal().hover();
```
Comment (labo-flg) [inline opencti-platform/opencti-front/tests_e2e/dashboard/createDashboard.spec.ts]: probably remove the comment here

File: opencti-platform/opencti-front/tests_e2e/dashboard/updateDashboard.spec.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ test('Create a new dashboard page and test update', async ({ page }) => {
   const dashboardDetailsPage = new DashboardDetailsPage(page);
   const dashboardForm = new DashboardFormPage(page);
   await page.goto('/dashboard/workspaces/dashboards');
-  await dashboardPage.openButtonModal().hover();
+  // await dashboardPage.openButtonModal().hover();
```
Comment (labo-flg) [inline opencti-platform/opencti-front/tests_e2e/dashboard/updateDashboard.spec.ts]: same

File: opencti-platform/opencti-front/tests_e2e/investigations/updateInvestigation.spec.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import InvestigationsPage from 'tests_e2e/model/investigations.pageModel';
+import InvestigationDetailsPage from '../model/investigationDetails.pageModel';
+import InvestigationsFormPage from '../model/form/investigationsForm.pageModel';
+
+test('Create a new dashboard page and test update', async ({ page }) => {
+  const investigationsPage = new InvestigationsPage(page);
+  const investigationDetailsPage = new InvestigationDetailsPage(page);
+  const investigationsForm = new InvestigationsFormPage(page);
+  await page.goto('/dashboard/workspaces/investigations');
+  // await dashboardPage.openButtonModal().hover();
```
Comment (labo-flg) [inline opencti-platform/opencti-front/tests_e2e/investigations/updateInvestigation.spec.ts:11]: same

**Review by labo-flg — DISMISSED** [review]
Thanks for your contribution!
I've done a quick review, you can also add the same remarks as in #7256.

File: opencti-platform/opencti-front/tests_e2e/dashboard/createDashboard.spec.ts (hunk 1 above)
Comment (CelineSebe) [reply opencti-platform/opencti-front/tests_e2e/dashboard/createDashboard.spec.ts]: Thanks a lot for your contribution. 
We have completly reworked this tests [dashboard.spec.ts:](https://github.com/OpenCTI-Platform/opencti/blob/master/opencti-platform/opencti-front/tests_e2e/dashboard/dashboard.spec.ts)
Could you remove it?

File: opencti-platform/opencti-front/tests_e2e/dashboard/updateDashboard.spec.ts (hunk 2 above)
Comment (CelineSebe) [reply opencti-platform/opencti-front/tests_e2e/dashboard/updateDashboard.spec.ts]: same here

**Review by Kedae — APPROVED** [review]

**Pull request merged**